src/main/_docs/admin-guide/admin-managing.md

```diff
@@ -130,6 +130,32 @@ However, when you do a `codenvy backup`, we do copy the Postgres data from the c
 # Migration
 It is possible to migrate your configuration and user data from a puppet-based installation of Codenvy (5.0.0-M8 and earlier) to the Dockerized version of Codenvy. Please contact our support team for instructions.
 
+# Auditing
+The Codenvy audit service provides information on historic licensing changes (moves between Fair Source and paid licenses for example) as well as the current state of the system with regards to license compliance and workspace ownership across all accounts.
+
+A system administrator can generate an audit report at any time:
+1. Log in with administrator credentials.
+2. Navigate in your browser to: `<Codenvy host url>/api/audit`.
+3. A file named `<date-time>.txt` will be downloaded.
```

Comment:
@bmicklea: my fault. Please, fix name of file in response of audit service on **report_\<datetime>.txt**